css/layout.css

```diff
@@ -2,33 +2,215 @@
   display: none !important;
 }
 
-.container {
-  max-width: 575px;
-  margin: 0 auto;
-  padding: 0 1.5rem;
+#container {
+  max-width: 860px;
+  margin-left: auto;
+  margin-right: auto;
 }
 
 .row {
   display: flex;
   flex-wrap: wrap;
+  flex-basis: 100%;
+  margin-top: -10px;
+}
+
+#label-url {
+  margin-left: 30px;
+  margin-top: 10px;
+}
+
+header {
+  width: 100%;
+  height: 0;
+}
+
+header p {
+  position: relative;
+  bottom: 100px;
+  right: 10px;
+  color: white;
+  font-size: 38px;
```

Comment:
Whenever possible, use `rem` for font sizes and other properties that should scale the same way (ex: `padding`, `border`, and `margin` on elements with text content): https://lms.learningfuze.com/code-guides/Learning-Fuze/curriculum/effective-css